src/containers/org/OrgContainer.js

```diff
@@ -29,46 +26,45 @@ import { useDispatch, useSelector } from 'react-redux';
 import type { Organization, UUID } from 'lattice';
 import type { RequestState } from 'redux-reqseq';
 
+import CollaborationsParticipationContainer from './CollaborationsParticipationContainer';
+import OrgDataSetsContainer from './OrgDataSetsContainer';
 import { DELETE_EXISTING_ORGANIZATION } from './actions';
-import { DataSetSearchResultCard, OrgActionButton } from './components';
+import { OrgActionButton } from './components';
 
 import { BadgeCheckIcon } from '../../assets';
 import {
   CrumbLink,
-  Flip,
   GapGrid,
-  SearchForm,
   SpaceBetweenGrid,
-  Spinner,
   StackGrid,
 } from '../../components';
-import { APPS, ES_FLAG_TYPE_RS_OPTIONS } from '../../core/edm/constants';
+import { APPS } from '../../core/edm/constants';
 import { resetRequestStates } from '../../core/redux/actions';
-import { ORGANIZATIONS, SEARCH } from '../../core/redux/constants';
+import { COLLABORATIONS, ORGANIZATIONS, SEARCH } from '../../core/redux/constants';
 import {
+  selectCollaborationsByOrgId,
   selectIsAppInstalled,
   selectOrganization,
-  selectSearchHits,
-  selectSearchPage,
-  selectSearchQuery,
-  selectSearchTotalHits,
 } from '../../core/redux/selectors';
 import { Routes } from '../../core/router';
 import {
   SEARCH_ORGANIZATION_DATA_SETS,
-  clearSearchState,
   searchOrganizationDataSets,
 } from '../../core/search/actions';
 import { MAX_HITS_10 } from '../../core/search/constants';
-import type { ReactSelectOption } from '../../types';
 
 const { PURPLE } = Colors;
 const { isNonEmptyString } = LangUtils;
 const {
-  isPending,
   isStandby,
   isSuccess,
 } = ReduxUtils;
+const { GET_COLLABORATIONS_WITH_ORGANIZATION, getCollaborationsWithOrganization } = CollaborationsApiActions;
+
+const StyledTabPanel = styled.div`
+  display: ${(props) => (props.show ? 'block' : 'none')};
+  padding: 8px 0 0;
+`;
```

Comment:
I imagine this will be used in other places. Why not move it to `src/components`?